src/sentry/assistant/guides.py

```diff
@@ -6,33 +6,87 @@
 GUIDES = {
     'issue': {
         'id': 1,
-        'page': 'issue',
-        'cue': _('Click here for a tour of the issue page'),
+        'cue': _('Get a tour of the issue page'),
         'required_targets': ['exception'],
         'steps': [
             {
-                'title': _('1. Stacktrace'),
+                'title': _('Stacktrace'),
                 'message': _(
-                    'See which line in your code caused the error and the entire call '
-                    'stack at that point.'),
+                    'See the sequence of function calls that led to the error, and global/local '
+                    'variables for each stack frame.'),
                 'target': 'exception',
             },
             {
-                'title': _('2. Breadcrumbs'),
+                'title': _('Breadcrumbs'),
                 'message': _(
-                    'See the events that happened leading up to the error, which often provides '
-                    'insight into what caused the error. This includes things like HTTP requests, '
-                    'database calls, and any other custom data you record. Breadcrumbs integrate '
-                    'seamlessly with many popular web frameworks and record .'),
+                    'Breadcrumbs are a trail of events that happened prior to the error. They\'re '
+                    'similar to traditional logs but can record more rich structured data. '
+                    'When Sentry is used with web frameworks, breadcrumbs are automatically '
+                    'captured for events like database calls and network requests.'),
                 'target': 'breadcrumbs',
             },
             {
-                'title': _('3. Additional Data'),
+                'title': _('Tags'),
                 'message': _(
-                    'Send custom data with every error, and attach tags to them that you can '
-                    'later search and filter by.'),
-                'target': 'extra',
+                    'Tags are arbitrary key-value pairs sent with every event. You can filter '
+                    'events by tags - for example, searching for all events from a specific '
+                    'machine, browser, release etc. The sidebar on the right shows you the '
+                    'distribution of tags for all events in this event group.'),
+                'target': 'tags',
+            },
+            {
+                'title': _('Resolution'),
+                'message': _(
+                    'Resolving an issue removes it from the default dashboard view of unresolved '
+                    'issues. You can ask Sentry to alert you when a resolved issue re-occurs.'),
```

Comment:
You should link them to account settings: https://sentry.io/account/settings/notifications/.